### ISSUE TITLE
regr losses can be used for classif?

### ISSUE THREAD

**Issue by berndbischl** [schalkdaniel/compboost#237]
shouldnt this raise an error?
cboost = Compboost$new(data = Sonar, target = "Class", loss = AbsoluteLoss)

**Comment by berndbischl** [comment]
this is fine for now, and i agree that the user should now what to do. 
OTOH: does compboost nowhere in the code "detects" that we are now doing classification? 
and has special code for this? what about posterior probabilies? is that supported?

**Comment by ja-thomas** [comment]
Only in the constructor.

We currently can only predict scores, each loss function should have a link/response function which is called in `compboost$predict` to transform the scores to props (or other values, i.e. `exp()` for gamma etc. just like in a glm). But this should be added on C++ side to the loss class.

@schalkdaniel: I think we created an issue for that? Or at least discussed it, but I cannot find the issue.

**Comment by ja-thomas** [comment]
Ah ok, there is a `responseTransformation` member function, so it's already implemented on c++ side, and only the call in R is missing?

**Comment by schalkdaniel** [comment]
Nope, should be there. It's like Janek said, each loss has a response function which transforms the scores. the default is `FALSE`.

**Comment by berndbischl** [comment]
conversation was helpful here, issue not important. you can close

**Comment by ja-thomas** [comment]
ok I was just blind, everything is already there :)
